hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -234,8 +239,8 @@ class HassioAddonInfo extends LitElement {
             <ha-chip
               hasIcon
               class=${classMap({
-                green: [5, 6].includes(Number(this.addon.rating)),
-                yellow: [3, 4].includes(Number(this.addon.rating)),
+                green: [6, 7, 8].includes(Number(this.addon.rating)),
+                yellow: [3, 4, 5].includes(Number(this.addon.rating)),
```

Comment:
This yellow could be a bit darker. It's hard to read with white text.

Reply:
It's using this 
https://github.com/home-assistant/frontend/blob/3a664d45a943cd726dc578aa5dbc4707ad6e27d8/src/resources/ha-style.ts#L53-L58

Reply:
alright let's keep it for now.